src/synchrotron.jl

```diff
@@ -57,18 +57,18 @@ function S_syn(ϵ, mps)
     # doppler factor D(Γ,θ)
     # Γ is the Bulk Lorentz Factor
     # Θ is the angle between the direction of the blob's motion and the direction to the observer
+    # B = 1 / β in the Dermer e tal. (1997) paper
     β = sqrt(1.0 - 1.0/mps.Γ^2)
     μ_obs = cos(mps.θ)
-    D = 1 / (mps.Γ*(1-(β * μ_obs)))
+    D = 1.0 / (mps.Γ*(1 - μ_obs/β))
     
     # Luminosity Distance dL(z)
-    dL = (2.0*mps.c / mps.Ho) * (mps.z+1.0 - sqrt(mps.z+1.0))
+    # Convert H_0 km / s / Mpc to cm / s / cm (cgs)
+    dL = (2.0*mps.c / (mps.Ho * 1.0E5 / 3.086E24 )) * (mps.z+1.0 - sqrt(mps.z+1.0))
```

Comment:
Need to convert H_0 into cgs units. You can check that the luminosity distance is similar to that on NED, for example.